core/src/main/java/io/questdb/Bootstrap.java

```diff
@@ -350,6 +350,8 @@ private void reportValidateConfig() {
             verifyFileSystem(path, cairoConfig.getSqlCopyInputRoot(), "sql copy input");
             verifyFileSystem(path, cairoConfig.getSqlCopyInputWorkRoot(), "sql copy input worker");
             verifyFileOpts(path, cairoConfig);
+            cairoConfig.getVolumeDefinitions().forEach((alias, volumePath) ->
+                    verifyFileSystem(path, volumePath, "create table allowed volume [" + alias + ']'));
```

Comment:
Can we verify File System types of the volumes? It can be non ext4 and we will have hard time to understand what is going on.

Reply:
I don't understand the question. At present verifyFileSystem is applyed to every path QuestDB has access to, to report what it is. This log will report the same way. Perhaps in documentation we should say that volumes need to use a file system readable by QuestDB?